src/wp-includes/post.php

```diff
@@ -7795,6 +7796,31 @@ function _prime_post_caches( $ids, $update_term_cache = true, $update_meta_cache
 	}
 }
 
+/**
+ * Prime post parent caches.
+ *
+ * @global wpdb $wpdb WordPress database abstraction object.
```

Comment:
```suggestion
 * @since 6.4.0
 *
 * @global wpdb $wpdb WordPress database abstraction object.
```

Add missing `@since` annotation.